redbot/cogs/mod/kickban.py

```diff
@@ -300,6 +301,11 @@ async def show_results():
             if errors:
                 text += _("\nErrors:\n")
                 text += "\n".join(errors.values())
+            if upgrades:
+                text += _(
+                    "\nFollowing user IDs have been upgraded from a temporary to a permanent ban:\n"
+                )
+                text += "\n".join(upgrades.values())
```

Comment:
It would probably be better to use a humanised list (`humanize_list()` util) here, rather than put each user ID in separate line.